vulnerability-rating-taxonomy.json

```diff
@@ -1,6 +1,6 @@
 {
   "metadata": {
-    "release_date": "2019-04-15T18:00:00+00:00"
+    "release_date": "2019-09-20T20:00:00+00:00"
```

Comment:
Kinda weird to be changing the time as well.  Though it is arbitrary, maybe if you're changing it change it to `00`.